DisCatSharp/Entities/Interaction/DiscordInteractionModalBuilder.cs

```diff
@@ -80,8 +80,8 @@ public DiscordInteractionModalBuilder WithCustomId(string customId)
         /// <param name="components">The components to append. Up to five.</param>
         /// <returns>The current builder to chain calls with.</returns>
         /// <exception cref="System.ArgumentException">Thrown when passing more than 5 components.</exception>
-        public DiscordInteractionModalBuilder AddModalComponents(params DiscordComponent[] components)
-            => this.AddModalComponents((IEnumerable<DiscordComponent>)components);
+        public DiscordInteractionModalBuilder AddModalComponents(params DiscordTextComponent[] components)
+            => this.AddModalComponents((IEnumerable<DiscordTextComponent>)components);
```

Comment:
shouldnt AddTextComponents be responsible for that?

Reply:
Overload to add multiple text components